buildSrc/subprojects/profiling/profiling.gradle.kts

```diff
@@ -1,5 +1,5 @@
 dependencies {
-    implementation("me.champeau.gradle:jmh-gradle-plugin:0.4.7")
+    implementation("me.champeau.gradle:jmh-gradle-plugin:0.4.8")
```

Comment:
Is the `jmh-gradle-plugin` update related to the failures?

Reply:
Yes, fixing this error required an upgrade.